aif360/algorithms/preprocessing/optim_preproc_helpers/data_preproc_functions.py

```diff
@@ -10,7 +10,7 @@ def custom_preprocessing(df):
         """
 
         # Group age by decade
-        df['Age (decade)'] = df['age'].apply(lambda x: x//10*10)
+        df['Age (decade)'] = df['age'].astype(int).apply(lambda x: x//10*10)
```

Comment:
Can we group integers or it is only feasible to group str?

Reply:
The 'age' had a String datatype due to which it was typecasted to integer and grouped as a decade.